src/index.ts

```diff
@@ -92,7 +120,15 @@ const parser = (html: string, options: Options = {}): Node[] => {
   }
 
   function onopentag(tag: string, attrs: Attributes) {
-    const buf: Node = {tag};
+    const start = getLoc(parser.startIndex);
+    const buf: NodeTag = {tag};
+
+    if (options.sourceLocations) {
+      buf.loc = {
```

Comment:
I think it will be not bad if we use the name for example `location`.

Reply:
Sure, if you prefer. `loc` is pretty commonly used e.g. in Babel and other parsers